kong/clustering/tls.lua

```diff
@@ -230,4 +232,25 @@ function tls.validate_client_cert(kong_config, cp_cert, dp_cert_pem)
 end
 
 
+--- Extract certificate details from the data plane certificate.
+---
+---@param dp_cert_pem string    # data plane cert text
+---
+---@return table? cert_details  # certificate details
+function tls.extract_dp_cert(dp_cert_pem)
```

Comment:
Would you mind adding a unit test for this function? Seems like somewhere in `spec/01-unit/19-hybrid` would be a good place.

Reply:
add